src/bundle.ts

```diff
@@ -93,7 +94,15 @@ const bundle = (document: unknown, bundleRoot: JsonPath, errorsRoot: JsonPath) =
         // Ignore invalid $refs and carry on
         if (!_path || !inventoryPath || !inventoryRef) return;
 
-        const bundled$Ref = get(document, _path);
+        let bundled$Ref: unknown;
+        if (typeof document === 'object' && document !== null) {
+          try {
+            bundled$Ref = resolveInlineRef(Object(document), $ref);
```

Comment:
Nice, I didn't feel like figuring out typings in my refs pr, but now I'll sneak a fix for this.